src/maus/__init__.py

```diff
@@ -105,7 +109,7 @@ def group_lines_by_segment_group(
                 this_sg = list(sg_group)
                 sg_draft = SegmentGroup(
                     discriminator=segment_group_key,  # type:ignore[arg-type] # might be None now, will be replace later
-                    ahb_expression=this_sg[0].ahb_expression or "",
+                    ahb_expression=this_sg[0].ahb_expression or "",  # segment groups barely ever have an expression
```

Comment:
Ist das weil sich die AHB Expression nicht wiederholt innerhalb der Segment Gruppe?

Reply:
das ist, weil die segmentgruppen im AHB initial oft (wenn sie nicht zum ersten mal genannt werden) sehr blank und alleine dastehen.
![grafik](https://user-images.githubusercontent.com/23094997/155756718-bce98201-534b-4a58-b037-74b05813b472.png)

Beim Eröffnen der SG5 hat sie noch ein "Kann", in der Kommunikationsverbindu-ng dann aber nicht mehr.